spark/src/test/scala/geotrellis/spark/tiling/ZoomedLayoutSchemeSpec.scala

```diff
@@ -9,7 +9,7 @@ import org.scalatest._
 class ZoomedLayoutSchemeSpec extends FunSpec with Matchers {
   describe("ZoomedLayoutScheme") { 
     it("Cuts up the world in two for lowest zoom level") {
-      val LayoutLevel(_, tileLayout) = ZoomedLayoutScheme().levelFor(1)
+      val LayoutLevel(_, tileLayout) = ZoomedLayoutScheme().levelForZoom(LatLng.worldExtent, 1)
```

Comment:
This would address the `ProjectedExtent` overload issue as well, those would go away.